README.md

```diff
@@ -61,16 +77,16 @@ If script finds any parser error, it stops immediately. In case of other error,
 
 If everything is fine, rule returns with 0.
 
-Normally, you should run the script:
+Normally, you should run the script (from `coreruleset` directory):
 
 ```
-crs-linter -r crs-setup.conf.example -r rules/*.conf
+../crs-linter/src/crs_linter/cli.py --debug -r crs-setup.conf.example -r 'rules/*.conf' -t util/APPROVED_TAGS -f ../crs-linter/FILENAME_EXCLUSIONS -v "4.13.0-dev"
```

Comment:
```suggestion
../crs-linter/src/crs_linter/cli.py \
  --debug \
  -r crs-setup.conf.example \
  -r 'rules/*.conf' \
  -t util/APPROVED_TAGS \
  -f ../crs-linter/FILENAME_EXCLUSIONS \
  -v "4.13.0-dev"
```

Reply:
This is a temporary example, we should add `FILENAME_EXCLUSIONS` to CRS too, then we should change here the path.